src/pages/index.js

```diff
@@ -85,9 +140,19 @@ viewImagePopup.setEventListeners();
 
```

Comment:
Needs correction: You should update the avatar when initializing the user. When I update the avatar and reload the page it becomes default